src/test/java/com/learning/pages/google/FirstSeleniumTest.java

```diff
@@ -3,23 +3,23 @@
 import static org.testng.Assert.assertEquals;
 import static org.testng.Assert.assertNotNull;
 
-import com.google.inject.Inject;
-import com.learning.ApplicationModule;
-import org.testng.annotations.Guice;
+import com.learning.TestsConfiguration;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
 import org.testng.annotations.Test;
 
-@Guice(modules = ApplicationModule.class)
-public class FirstSeleniumTest {
+@ContextConfiguration(classes = TestsConfiguration.class)
+public class FirstSeleniumTest extends AbstractTestNGSpringContextTests {
 
-  @Inject
+  @Autowired
   private GoogleSearchPage googleSearchPage;
 
   @Test
-  void shouldGooglePageNotBeNullWhenInjected() {
+  void shouldGooglePageNotBeNullWhenAutowired() {
```

Comment:
Nice one adding this test, but it deserves to be in a different test class, since it's not actually a feature/system test.